dojo/tools/aws_prowler_v3/parser.py

```diff
@@ -17,15 +17,92 @@ def get_label_for_scan_types(self, scan_type):
         return AWSProwlerV3Parser.SCAN_TYPE[0]
 
     def get_description_for_scan_types(self, scan_type):
-        return "Export of AWS Prowler JSON V3 format."
+        return "Exports from AWS Prowler v3 in JSON format or from Prowler v4 in OCSF-JSON format."
 
     def get_findings(self, file, test):
-        if file.name.lower().endswith('.json'):
+        if file.name.lower().endswith('.ocsf.json'):
```

Comment:
Could you please move the functionalities of v3 and v4 into separate files?

Reply:
Why? What's the benefit in separating them? Other scanners do the same in the same manner.

Reply:
Separate files for separate formats/extension is the standard the project is moving towards, largely thanks to @manuel-sommer

Reply:
Ok, I just separated them.